config/config.go

```diff
@@ -144,6 +144,8 @@ type Config struct {
 
 	// TODO: It will soon be discarded in the near future
 	DevSnapSync bool `long:"devsnapsync" description:"Enable snap sync for P2P that only exist in development mode"`
+
+	SnapTimeout int `long:"snaptimeout" description:"Unable to find a peer that supports snap-sync service for setting IBD timeout(seconds), it will switch to the normal sync method"`
```

Comment:
`SnapTimeout` -> `FindSnapSyncPeerTimeout`
a better description, `During the IBD phase, if no peer node with snap-sync service enabled is found after the timeout (seconds), the node will switch to using the normal sync method.`

Reply:
or `NoSnapSyncPeerTimeout` is also OK for me

Reply:
https://github.com/Qitmeer/qng/pull/848/commits/a9bcdf4c8882bc4a69b6b50928a817cd7be46a2b